browser/ephemeral_storage/brave_ephemeral_storage_service_delegate.cc

```diff
@@ -144,4 +152,39 @@ void BraveEphemeralStorageServiceDelegate::OnBrowserAdded(Browser* browser) {
 }
 #endif
 
+void BraveEphemeralStorageServiceDelegate::CloseTabsForDomainAndSubdomains(
+    content::WebContents* contents,
```

Comment:
I feel this shouldn't accept `WebContents`, but rather act on browser `Profile` level.